### PR TITLE
Include filename in babel/babylon errors

### PR DESCRIPTION
**Pull request by isaacs**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -62,3 +62,15 @@ test.serial('disable cache', t => {
 	importJsx(fixturePath('react'), {cache: false});
 	t.true(babel.transform.calledTwice);
 });
+
+test('syntax error includes filename', t => {
+	const file = fixturePath('syntax-error.js');
+	let threw = false;
+	try {
+		importJsx(file);
```
Comment (vadimdemedes) [inline test/test.js]: Could you replace that with this?

```js
const error = t.throws(() => importJsx(file));
t.is(error.message.split(':')[0], file);
```

**Comment by vadimdemedes** [comment]
Thanks!

File: test/test.js (hunk 1 above)
Comment (isaacs) [reply test/test.js]: Done!

**Comment by vadimdemedes** [comment]
Published 1.3.1 with this, great improvement!

**Pull request merged**